app/controllers/dashboard_developers_controller.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
 class DashboardDevelopersController < ApplicationController
-  def index; end
+  authorize_resource :class => :DashboardDevelopersController
```

Comment:
kindly use the updated syntax for hash `class: :DashboardDevelopersController`

Reply:
please review code again.